packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1718,12 +1767,24 @@ export class ContainerRuntime
 			}
 			return provideEntryPoint(this);
 		});
+
+		// If we loaded from pending state, then we need to skip any ops that are already accounted in such
```

Comment:
It's not clear to me what this means. Are there ops (stashed) in pending state that needs to be skipped? And skipped from what?